app/javascript/mastodon/components/common_counter.js

```diff
@@ -0,0 +1,62 @@
+// @ts-check
```

Comment:
This annotation used to enable TypeScript checks in the JavaScript files. In JavaScript annotations are taken from JSDocs. Enabling TypeScript checks also allows for quick renaming and other helpful functions. Probably harmless by itself, when TS is not available.